ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -665,7 +665,8 @@ def sync_time(host, server):
 
     host.run_command(['systemctl', 'stop', 'chronyd'])
     host.run_command(['chronyd', '-q',
-                      "server {srv} iburst".format(srv=server.hostname)])
+                      "server {srv} iburst".format(srv=server.hostname),
+                      'pidfile /tmp/chronyd.pid', 'bindcmdaddress /'])
```

Comment:
This will leave ``/tmp/chronyd.pid`` around after the command exits. Is this acceptable?

Reply:
And do we want a predictable file name?

Reply:
I haven't thought of that. We can probably assume that chronyd creates the pid file in a secure way.

Also it's just for integration tests. Should we care?

Reply:
Yeah, for tests perhaps not a big deal. I guess the question is, will the presence of an existing pid file cause issues? I suppose if the tests pass that will tell us.

Reply:
I tried it on a test machine, no error:
```
# chronyd -q "server de.pool.ntp.org iburst" "pidfile /tmp/chronyd.pid" "bindcmdaddress /"
2018-05-02T13:45:27Z chronyd version 3.3 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP +SCFILTER +SIGND +ASYNCDNS +SECHASH +IPV6 +DEBUG)
2018-05-02T13:45:27Z Initial frequency -10.784 ppm
2018-05-02T13:45:31Z System clock wrong by -0.002465 seconds (step)
2018-05-02T13:45:31Z chronyd exiting
# cat /tmp/chronyd.pid 
46526
# chronyd -q "server de.pool.ntp.org iburst" "pidfile /tmp/chronyd.pid" "bindcmdaddress /"
2018-05-02T13:45:32Z chronyd version 3.3 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP +SCFILTER +SIGND +ASYNCDNS +SECHASH +IPV6 +DEBUG)
2018-05-02T13:45:32Z Initial frequency -10.784 ppm
2018-05-02T13:45:36Z System clock wrong by 0.001738 seconds (step)
2018-05-02T13:45:36Z chronyd exiting
# cat /tmp/chronyd.pid 
46547
```

Reply:
We could of course do it using created temporary file with random name and have it deleted after.

Or use `#restorecon -Rv /var/run/`.